tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dynamic-page-validator.cy.js

```diff
@@ -0,0 +1,53 @@
+import { selfAssessmentSlug } from "../../helpers/page-slugs.js";
+import DataMapper from "export-processor/data-mapper.js";
+import { validateSections } from "./dpv-validators/section-validator.js";
+import { validateNonAuthorisedPages } from "./dpv-validators/non-auth-pages-validator.js";
+import { validateNavigationLinks } from "./dpv-validators/nav-links-validator.js";
+import { validateRecommendationForMaturity } from "./dpv-validators/recommendations-validator.js";
+import { validateSelfAssessmentPageOnStart } from "./dpv-validators/self-assessment-page-validators.js";
+
+const dataMapper = new DataMapper(require('../../fixtures/contentful-data'));
+
+describe("Navigation links and non-authorised pages", () => {
```

Comment:
It looks a bit ugly, but I think you can remove all the other `dataLoaded` checks if you add one here, by wrapping all the tests in one if statement. E.g.
```suggestion
const dataMapper = new DataMapper(require('../../fixtures/contentful-data'));

if (dataLoaded(dataMapper)) {
    describe("Navigation links and non-authorised pages", () => {
    //Rest of the code here
    }
    
    //Rest of the tests
}
```

Reply:
Nevermind I lied ignore me :+1:. There should be a better way though yours is great and works so 💯